index.html

```diff
@@ -1,32 +1,164 @@
+                   <li class="frame counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame year">2015</li>
+                 </ul>
+                 <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+           <div class="card card3">
+            <div class="pic1"><img src="/img/Snapshoot Portfolio (3).png" alt="Snapshoot"></div>
+            <img src="/img/Project Title (1).svg" alt="" class="title">
+                 <ul class="frame-container">
+                   <li class="frame client">CANOPY</li>
+                   <li class="frame Counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame role">Back End Dev</li>
+                   <li class="frame counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame year">2015</li>
+                 </ul>
+                 <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+           <div class="card card4">
+            <div class="pic1"><img src="/img/Snapshoot Portfolio (4).png" alt="Snapshoot"></div>
+            <img src="/img/Project Title(2).svg" alt="" class="title">
+                 <ul class="frame-container">
+                   <li class="frame client">CANOPY</li>
+                   <li class="frame Counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame role">Back End Dev</li>
+                   <li class="frame counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame year">2015</li>
+                 </ul>
+                 <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="contact-container">
+        <div class="one">
+         <contact class="about"><img src="/img/Primary Text.svg" alt="" class="about-pic"></contact>
```

Comment:
- [ ] Similarly, for the about section, kindly use a header tag, `hx`, instead of an image. Using header tag for the project title ensures accessibility and SEO ✨⛏